com.unity.render-pipelines.high-definition/Documentation~/Dynamic-Resolution.md

```diff
@@ -1,13 +1,17 @@
 # Dynamic resolution
 
-Dynamic resolution reduces the workload on the GPU, which helps maintain a stable target frame rate. The High Definition Render Pipeline (HDRP) uses dynamic resolution to lower the resolution of the render targets that the main rendering passes use. To do this, HDRP uses hardware dynamic resolution, if the platform supports it, otherwise it uses a software version. The main difference is that, for hardware dynamic resolution, the hardware treats the render targets up until the back buffer as being of the scaled size. This means that is is faster to clear the render targets.
+Dynamic resolution reduces the workload on the GPU and maintains a stable target frame rate. The High Definition Render Pipeline (HDRP) uses dynamic resolution to lower the resolution of the render targets that the main rendering passes use.
 
-Hardware Dynamic resolution is supported on:
+HDRP uses hardware dynamic resolution automatically if your platform supports it. Otherwise, HDRP uses software dynamic resolution. Hardware dynamic resolution treats the render targets, up until the back buffer, as if they are all the scaled size. This means HDRP clears the render targets faster.
```

Comment:
> HDRP uses hardware dynamic resolution automatically if your platform supports it. 

This is not true, it is just the default option. When that option is selected, but not available it fallbacks to software.

Reply:
I think the original wording was bad as well :)